COMPONENTS_DOCUMENTATION_GUIDELINES.md

```diff
@@ -0,0 +1,110 @@
+# Components documentation best practices
+This guide provides essential best practices for documenting components using Storybook. 
+This doc is still work in progress and will constantly be updated.
+
+1. Always use live component instances instead of displaying Vibe's components as pictures. Pictures will not be updated according to design changes, theme changes, and bug fixes.
+2. Keep the examples in the stories minimal, without using unnecessary props that don't serve the purpose of demonstrating the story unless the Vibe's designers specifically request it.  For example, this minimal story does not contain not necessary props:
+```js
+<Canvas>
+    <Story name="Sizes">
+        <Button size={Button.sizes.LARGE}>Large</Button>
+        <Button size={Button.sizes.MEDIUM}>Medium</Button>
+        <Button size={Button.sizes.SMALL}>Small</Button>
+    </Story>
+</Canvas>
+```
+
+3. Whenever possible, avoid adding titles to the variants displayed within the story to maintain clear and simpler story code. For example, instead of writing this:
+```js
+<Canvas>
+  <Story name="Sizes">
+      <Flex direction={Flex.directions.ROW} gap={Flex.gaps.SMALL}>
+          <StoryDescription description="Large" vertical align={Flex.align.START}>
+              <Button size={Button.sizes.LARGE}>Large</Button>
+          </StoryDescription>
+          <StoryDescription description="Medium" vertical align={Flex.align.START}>
+              <Button size={Button.sizes.MEDIUM}>Medium</Button>
+          </StoryDescription>
+          <StoryDescription description="Small" vertical align={Flex.align.START}>
+              <Button size={Button.sizes.SMALL}>Small</Button>
+          </StoryDescription>
+      </Flex>
+  </Story>
+</Canvas>
+```
+Do this:
+```js
+<Canvas>
+    <Story name="Sizes">
+        <Button size={Button.sizes.LARGE}>Large</Button>
+        <Button size={Button.sizes.MEDIUM}>Medium</Button>
+        <Button size={Button.sizes.SMALL}>Small</Button>
+    </Story>
+</Canvas>
+```
+
+4. When laying out components, utilize Vibe's Flex component instead of implementing your own flex layout using CSS. Refer to the code example below for clarification. **Please use Flex only when displaying multiple examples** in a story and only if the default story layout and gaps do not match the documentation design without using Flex as a wrapper.
```

Comment:
```suggestion
4. The default `<Story/>` layout provides a solution for both a single component example and multiple components example. In case of multiple components example where the `<Story/>` layout is insufficient, you may use **only** Vibe's `<Flex/>` layout component. Any other custom layout solution is forbidden. 
```